docs/development.rst

```diff
@@ -7,12 +7,10 @@ This section dives into the development process of *airflow-dbt-python* by descr
 
 If you would like to dig a bit into the internals of *airflow-dbt-python* and learn about the development philosophy, it's recommended that you check out the :ref:`how_does_it_work` documentation.
 
-Project management: Poetry
+Project management: *uv*
```

Comment:
Parts of this doc are somewhat out of date. I've done some changes here, but a few more are required, I think.

Reply:
A lot of the pain that came with resolving dependencies really goes away with uv.